documentation/src/main/asciidoc/topics/ref_server_realm_trust.adoc

```diff
@@ -1,18 +1,26 @@
-[id='trust_store_realm-{context}']
+[id='trust-store-realms_{context}']
 = Trust Store Realms
-Trust store realms use keystores that contain certificates, or certificate chains, that clients must present to connect to {brandname} Server.
+{brandname} Server can encrypt traffic using keystores that contain server and client certificates.
```

Comment:
can encrypt traffic using industry standard SSL/TLS protocols. Such protocols require the use of server and (optionally) client certificates which must be provided via keystores.